userguide/tutorials/plugin_use_cases.adoc

```diff
@@ -0,0 +1,87 @@
+This document includes some specific use cases related to plugins.
```

Comment:
Is this linked from anywhere yet? If not, you could add an entry in the [left nav](https://github.com/killbill/killbill-docs/blob/eb80e239677969eb37a2dc671d84766c96a64acd/html5/_main_toc.html.slim).